flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -335,8 +365,30 @@ public Builder<T> exposeLocality(boolean newExposeLocality) {
       return this;
     }
 
+    /**
+     * Set the read properties for Flink source. View the supported properties in {@link
+     * FlinkReadOptions}
+     */
+    public Builder<T> set(String property, String value) {
```

Comment:
do we need this one? I thought all options are exposed via setter methods in this class.

Reply:
I tend to keep it and give developers an alternative.

Reply:
sure. let's keep it. I am not too opinionated on this.